### ISSUE TITLE
Limit app to work only a specific device only

### ISSUE THREAD

**Issue by e4basil** [mik3y/usb-serial-for-android#522]
Is there any way to limit the app will support only a specific device only?

**Comment by kai-morich** [comment]
yes, see last sentence [here](https://github.com/mik3y/usb-serial-for-android#probing-for-unrecognized-devices), or use only a `CustomProber` with fixed VID/PID instead of `getDefaultProber`
